webapp/package.json

```diff
@@ -25,5 +26,5 @@
     "platform/types",
     "playbooks",
     "boards"
-  ]
+  ],
```

Comment:
This extra comma is actually causing the `npm install` to fail for this. Some JSON code allows for trailing commas, but strictly speaking, they're not supported in JSON

Reply:
It didn't fail on my PC and I guess the comma was added by the npm command or by VSCode maybe. 
I'll fix it anyway.